src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressElement.java

```diff
@@ -119,12 +138,21 @@ public boolean accept(AuditEvent event) {
      */
     private boolean isFileNameAndModuleNotMatching(AuditEvent event) {
         return event.getFileName() == null
-                || !fileRegexp.matcher(event.getFileName()).find()
+                || fileRegexp != null && !fileRegexp.matcher(event.getFileName()).find()
                 || event.getLocalizedMessage() == null
                 || moduleId != null && !moduleId.equals(event.getModuleId())
                 || checkRegexp != null && !checkRegexp.matcher(event.getSourceName()).find();
     }
 
+    /**
+     * Is matching by message.
+     * @param event event
+     * @return true is matching
+     */
+    private boolean isMessageNameNotMatching(AuditEvent event) {
```

Comment:
"not" in name is not good.
I see that you tried to be consistent with similar method, but let's not doing this, one day we will refactor it too.
One day we will have a Check for this.

Reply:
@romani This is issue #4979 . Please review the issue when you get a chance.

Reply:
approved.

Reply:
This was fixed.